doc/telescope.txt

```diff
@@ -583,6 +583,7 @@ builtin.git_branches({opts})                          *builtin.git_branches()*
       - `<C-r>`: rebases currently selected branch
       - `<C-a>`: creates a new branch, with confirmation prompt before creation
       - `<C-d>`: deletes the currently selected branch, with confirmation
+      - `<C-m>`: merges the currently selected branch, with confirmation
```

Comment:
You need to change this here: https://github.com/nvim-telescope/telescope.nvim/blob/master/lua/telescope/builtin/init.lua#L171-L179

doc/telescope.txt is 100% autogenerated

Reply:
Thanks I wasn't sure which one to update :)

Updated